UnityProject/Assets/Scripts/PlayGroups/PlayerMove.cs

```diff
@@ -243,7 +243,7 @@ private void InteractDoor(Vector3 currentPosition, Vector3 direction)
             // Attempt to open door
             if (doorController != null && allowInput)
             {
-                doorController.CmdCheckDoorPermissions(doorController.gameObject, this.gameObject);
+                PlayerManager.LocalPlayerScript.playerNetworkActions.CmdCheckDoorPermissions(doorController.gameObject, this.gameObject);
```

Comment:
Known solution, fast fix... Great!
Think you know how thorough your testing needs to be next overhaul you write ;)

Reply:
I would like it to use a seperate var for PNA though!

Reply:
The script didn't have a variable for it, but I can fix it.